js/coinbin.js

```diff
@@ -1231,6 +1267,58 @@ $(document).ready(function() {
 		});
 	}
 
+	/* retrieve unspent data from electrum proxy */
+	function listUnspentElectrum(redeem, api){
+		$.ajax ({
+			type: "GET",
+			url: `${api}address/${redeem.addr}/utxo`,
+			dataType: "json",
+			async: false,
+			error: function(data) {
+				console.log(`Error from ${api}address/1KFHE3232AswwryaoccDb6qcT6DbY1/utxo`);
```

Comment:
What's this address?

Reply:
@junderw Nice catch, that is why we need linters :(